api/v1beta1/flowcollector_types.go

```diff
@@ -532,7 +532,7 @@ type FlowCollectorConsolePlugin struct {
 	//+kubebuilder:validation:Minimum=1
 	//+kubebuilder:validation:Maximum=65535
 	//+kubebuilder:default:=9001
-	// port is the plugin service port
+	// port is the plugin service port. Do not use 9002, which is reserved for metrics.
```

Comment:
Note: I didn't want to add a new port config to the CRD, I think it brings very little value if any, while bloating the CRD with new configs ...

Reply:
@jotak while we're at it, do you think it would be good idea to address this: https://issues.redhat.com/browse/NETOBSERV-511 ?